docs/source/market_config.rst

```diff
@@ -140,3 +162,5 @@ Due to the configuration of the market opening frequency and duration, the timet
       Delivery CRM           :crm02, 2019-01-01 01:00, 4h
       Bidding CRM            :crm03, 2019-01-01 04:00, 30m
       Delivery CRM           :crm04, 2019-01-01 05:00, 4h
+
+$PLACEHOLDER$ This is the code snippet that describes the market configuration options and provides an example configuration for an EOM and CRM market.
```

Comment:
What is meant with that placeholder?

Reply:
Very Good question... I did not add it on purpose...